padevents/padevents.py

```diff
@@ -912,13 +918,17 @@ def removeDailyReg(self, channel_id, server):
 
 
 class Event:
-    def __init__(self, scheduled_event: "DgScheduledEvent"):
+    def __init__(self, scheduled_event: "DgScheduledEvent", bot=None):
+        self.bot = bot
+        self.db_context = self.bot.get_cog("Dadguide").database
```

Comment:
can merge into 1 line:

`self.db_context: DbContext = self.bot.get_cog("Dadguide").database`

Reply:
you want just `self.db_context = bot.get_cog("Dadguide").database` ? (as written doesn't work)
everywhere else that requires a bot as an argument then assigns it as an attribute so I figured do that here too but it won't be used so I could change it to this yeah

Reply:
(changed it)